sdk/core/src/lib.rs

```diff
@@ -30,16 +34,19 @@ use oauth2::AccessToken;
 use std::fmt::Debug;
 use uuid::Uuid;
 
-pub use self::http::{Request, Response};
+pub use self::http::Response;
+pub use context::Context;
 pub use headers::AddAsHeader;
 pub use http_client::{to_json, HttpClient};
 pub use models::*;
-pub use policy::*;
+pub use request::*;
 
 pub type RequestId = Uuid;
 pub type SessionToken = String;
 pub const EMPTY_BODY: &[u8] = &[];
 
+type BoxedFuture<T> = Box<dyn std::future::Future<Output = policies::PolicyResult<T>> + Send>;
```

Comment:
If we're hardcoding `PolicyResult` maybe this should be inside the `policy` module

Reply:
Defintely! 👍